app/questionnaire/placeholder_parser.py

```diff
@@ -178,6 +180,23 @@ def _parse_transforms(
 
                 transform_args[arg_key] = resolved_value
 
+                if (
+                    transform["transform"] == "format_currency"
+                    and arg_value.get("source") == "answers"
```

Comment:
The source could be `previous_transform` which wouldn't currently work. This would be a tricky one, and not something we could easily handle as the transformed value could go through any number of transformation. Therefore, we might have to make it a an exception and document in the decision log and say format currency when used with previous transform won't take into account the schema limit.
The alternative would be to enforce author to set the decimal places explicitly. (We will likely do this, if this ever becomes a problem)

Reply:
Added to the decision log now 👍